src/components/Input/InputPair/index.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+
+import { ReactNode } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { IconDefinition, library, icon } from "@fortawesome/fontawesome-svg-core";
+import styles from "./inputpair.module.css";
+
+type InputPairProps = {
+	icon?: IconDefinition;
+	className?: string;
+	children: [ReactNode, ReactNode];
+};
+
+export default function InputPair({ className, ...props }: InputPairProps) {
+	
+	// If icon exists add to libary
+    if(props.icon)
+		library.add(props.icon);
```

Comment:
May be unnecessary, we already import the entire library

Reply:
Right, thanks